selfdrive/car/hyundai/fingerprints.py

```diff
@@ -1653,6 +1653,8 @@
       b'\xf1\x00MQ4HMFC  AT KOR LHD 1.00 1.12 99210-P2000 230331',
       b'\xf1\x00MQ4HMFC  AT USA LHD 1.00 1.10 99210-P2000 210406',
       b'\xf1\x00MQ4HMFC  AT USA LHD 1.00 1.11 99210-P2000 211217',
+      b'\xf1\x00MQ4HMFC  AT USA LHD 1.00 1.13 99210-P2000 230619',
+      b'\xf1\x00MQ4HMFC  AT KOR LHD 1.01 1.03 99210-P2500 230914',
```

Comment:
Why are there two dates?

Reply:
usa fw is -> Add Fingerprint for 2024 Kia Sorento Hybrid #30683

Reply:
Why would you add it here?